contracts/lib/SafeERC20.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
```

Comment:
What do you think of using the SafeERC20 from Zep instead of creating a new one here?

Reply:
You can install the zep lib 0.7 and leave this branch as 0.7 too instead of merging to master

Reply:
Not merging to master actually. Just create a PR so that people can view the changes wasily. 

Will be releasing versions.

Reply:
When are you planning to release the version? I'm blocked :(